### PR TITLE
Mac Installer fix

### LINKED ISSUE #68: Installation stays in /Application/Unity when installing on OSX
When installing Unity version from `u3d` the installation stays in `/Application/Unity` and requires additional `u3d` loop in order for installation to be moved to proper versioned directory even though it asks for the elevated permissions.

No errors or warning are shown during the installation.

### LINKED ISSUE #71: [Feature request] Installing package into existing installations
It would be great if I could add module/package to existing installation without removing it first.

(It caches the files, but what if the cache wouldn't be there, would I have to redownload everything?)

### PR DESCRIPTION
**Pull request by niezbop**
This fix makes sure that we move new Unity installs under our correct format (ie Unity_0.0.0x0) for fresh installs.

This should fix #68, and this being fixed would fix #71

### CODE REVIEW

**Comment by lacostej** [comment]
1. we use sanitize_install for Linux. Can't we do the same?
2. has this been tested in the case where we install modules?

**Comment by niezbop** [comment]
1. Not exactly the same, because if we were to put it in install method it would force sanitize on package installation wether the user wants it or not. But we could easily replace the FileUtils.move by a sanitize.

2. It hasn't been tested at all, I still do not have an easy access to a Mac. I was hoping you could!


_Sent from my HUAWEI DIG-L21HN using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: lib/u3d/installer.rb [hunk 1]
```diff
@@ -132,13 +132,16 @@ def install_pkg(file_path, version: nil, target_path: nil)
       target_path ||= DEFAULT_MAC_INSTALL
       command = "installer -pkg #{file_path.shellescape} -target #{target_path.shellescape}"
       unity = installed.find { |u| u.version == version }
+      temp_path = File.join(target_path, 'Applications', 'Unity')
       if unity.nil?
         UI.verbose "No Unity install for version #{version} was found"
         U3dCore::CommandExecutor.execute(command: command, admin: true)
+        destination_path = File.join(target_path, 'Applications', UNITY_DIR % version)
+        Utils.ensure_dir destination_path
+        FileUtils.mv temp_path, destination_path
```
Comment (lacostej) [inline lib/u3d/installer.rb]: I think this will not work the `ensure_dir` call will create
```
/Applications/Unity_x.x.x.x//Unity
```

Comment (niezbop) [reply lib/u3d/installer.rb]: Tested and it doesn't work as you expected it

**Comment by niezbop** [comment]
Changed to sanitize

**Comment by lacostej** [comment]
After your last comment, I think I preferred the previous version.

We have to revisit all this anyway because I think we should sanitize only at installation end. I.e. once when calling `install_modules`

**Comment by niezbop** [comment]
Sounds better indeed. I will do so then.

**Pull request merged**